### PR TITLE
ci: test server with PostGIS 3.2

### LINKED ISSUE #7753: Add PostGIS 3.2 to the CI
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

### Current Limitation

PostGIS 3.2 was recently [released](https://www.postgis.net/2021/12/18/postgis-3.2.0/). The CI is currently not tested against it to se if any bugs were introduced.

### Feature / Enhancement Description

Should always test against the latest.

### Example Use Case

N/A

### Alternatives / Workarounds

Use the latest without tests.

### 3rd Party References

N/A

### PR DESCRIPTION
**Pull request by cbaker6**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

### Issue Description

CI currently not testing against [PostGis 3.2](https://www.postgis.net/2021/12/18/postgis-3.2.0/). Close #7753

Related issue: #7753 

### Approach

Add PostGis 3.2 to the CI

### TODOs before merging

- [x] Add changes to documentation (guides, repository pages, in-code descriptions)
- [x] A changelog entry is created automatically using the pull request title (do not manually add a changelog entry)

### CODE REVIEW

**Comment by cbaker6** [comment]
This is ready for review. This will need repo setting changes to account for name changes to tests (3.1->3.2)...

**Comment by mtrezza** [comment]
> Will needs settings changes to account for updated tests...

What does that mean?

**Comment by cbaker6** [comment]
Here I was eluding to the "expected/required" tests to pass the CI. Since the 3.2 was added, someone on the PMC has to update the repo settings.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -187,13 +187,16 @@ jobs:
           - name: PostgreSQL 11, PostGIS 3.1
             POSTGRES_IMAGE: postgis/postgis:11-3.1
             NODE_VERSION: 16.13.0
-          - name: PostgreSQL 12, PostGIS 3.1
+          - name: PostgreSQL 11, PostGIS 3.2
+            POSTGRES_IMAGE: postgis/postgis:11-3.2
+            NODE_VERSION: 16.13.0
+          - name: PostgreSQL 12, PostGIS 3.2
             POSTGRES_IMAGE: postgis/postgis:12-3.1
```
Comment (mtrezza) [inline .github/workflows/ci.yml]: Is this correct? It says "PostGIS 3.2", but the image is  "-3.1". Also in some lines below.

Comment (cbaker6) [reply .github/workflows/ci.yml]: It's not correct, will update

**Review by mtrezza — APPROVED** [review]
Look good! Is this ready for merge once the CI passes?

**Comment by cbaker6** [comment]
Yup, it's ready

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-alpha.15](https://github.com/parse-community/parse-server/releases/tag/5.0.0-alpha.15)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-beta.10](https://github.com/parse-community/parse-server/releases/tag/5.0.0-beta.10)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.1.0](https://github.com/parse-community/parse-server/releases/tag/5.1.0)

**Pull request merged**